### ISSUE TITLE
[Repository] Sticky header of cache stat table is slowing down the page

### ISSUE THREAD

**Issue by jc7447** [jc7447/BetterDynAdmin#102]
We have a performance issue with setupCacheTableHeaderFixed, with this method it takes 500ms to setup the cache stat table for a repository with 60 descriptors , which is huge regarding the feature.

I have put the method in comment and now it takes 100ms and the feature seems to work fine (without sticky header).

@troussej may be we could setup the sticky header only when the table is displayed, or put this feature as optional, what do you think ?

**Comment by troussej** [comment]
hmm it shouldn't depend on the nb of descriptors. 

yeah deactivate it for now, and we'll see how to tune it.

**Comment by troussej** [comment]
ok so I found out that the jquery.width() method is very slow, because it causes a lot of box sizing recalculations.  http://blog.jquery.com/2012/08/16/jquery-1-8-box-sizing-width-csswidth-and-outerwidth/

I ended up redoing the whole thing with more css, but still have to call a single .width() that lasts ~90ms. 

The whole thing now takes 100ms... Maybe we can add a toggle mecanism, or I will calculate that only during the first scroll so that it doesn't slow the page loading.

I also made the whole table fit in the page width, so we don't have to scroll right, and made all column the same width (1 calculation), but the header is less readable.
